index.js

```diff
@@ -35,6 +37,12 @@ function getAssets() {
     return  _.cloneDeep(result);
 }
 
+function hash(path) {
```

Comment:
Maybe rename the `path` argument to `str`. Since that's all the `hash` function cares about